src/app/models/api-response.model.ts

```diff
@@ -0,0 +1,29 @@
+export abstract class ApiResponseModel {
+  protected readonly flowStatus: string;
+  protected readonly flowStatusMessage: string;
+  protected readonly apiResponse: any;
```

Comment:
Any way you can work on not using any?

Reply:
May be best to have individual types for each response and pass that through with whatever call is using the main call